api/client.go

```diff
@@ -360,6 +360,14 @@ func (c *Client) Embeddings(ctx context.Context, req *EmbeddingRequest) (*Embedd
 	return &resp, nil
 }
 
+func (c *Client) Template(ctx context.Context, req *TemplateRequest) (*TemplateResponse, error) {
```

Comment:
What are we templating? This is not a helpful name. Can we find a new one? Also, docs would be very helpful.